scripts/Interface/reduction_gui/reduction/diffraction/diffraction_adv_setup_script.py

```diff
@@ -288,8 +288,13 @@ def from_xml(self, xml_str):
             # Caching options
             # split it into the three cache dirs
             # NOTE: there should only be three entries, if not, let it fail early
-            self.cache_dir_scan_save, self.cache_dir_scan_1, self.cache_dir_scan_2 = BaseScriptElement.getStringElement(
-                instrument_dom, 'cachedir', default=";;").split(";")
+            try:
+                self.cache_dir_scan_save, self.cache_dir_scan_1, self.cache_dir_scan_2 = BaseScriptElement.getStringElement(
+                    instrument_dom, 'cachedir', default=";;").split(";")
```

Comment:
Parallel to the change above
```suggestion
                    instrument_dom, 'cachedir', default=";;").replace(";", ",").split(";")
```
which will maintain backwards compatibility